sdk/tables/azure-tables/src/index.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// This node built-in must be shimmed for the browser.
+
+// This is a node dependency that needs to be replaced with a
+// different implementation in the browser.
+
+
+// this is a utility function from a library that should be external
+// for both node and web
+import { isNode } from "@azure/core-http";
+
+export interface Foo {
```

Comment:
You can probably remove this interface and just leave the dummy helloWorld function.